fuzz.go

```diff
@@ -61,6 +63,33 @@ func NewWithSeed(seed int64) *Fuzzer {
 	return f
 }
 
+// NewFromGoFuzz is a helper function that enables using gofuzz (this
+// project) with go-fuzz (https://github.com/dvyukov/go-fuzz) for continuous
+// fuzzing. Essentially, it enables translating the fuzzing bytes from
+// go-fuzz to any Go object using this library.
+//
+// Example: use go-fuzz to test the function `MyFunc(int)` in the package
+// `mypackage`. Add the file: "mypacakge_fuzz.go" with the content:
+//
+// // +build gofuzz
+// package mypacakge
+// import "github.com/google/go-fuzz"
+// func Fuzz(data []byte) int {
+// 	var i int
+// 	fuzz.NewFromGoFuzz(data).Fuzz(&i)
+// 	MyFunc(i)
+// 	return 0
+// }
+func NewFromGoFuzz(data []byte) *Fuzzer {
+	maxint := big.NewInt(math.MaxInt64)
+	seed := big.NewInt(1)
+	for _, b := range data {
```

Comment:
btw, I think this loop is equivalent to just taking the last byte and making it an int64.

Reply:
How come?

Reply:
Sorry, I misread this. It is not equivalent to that :)

Reply:
(This does have the property that the order of the bytes doesn't matter; {1,2,3} and {3,2,1} produce the same seed. If we were to go with the approach of just collapsing to a single seed, I'd recommend interpreting all the bytes in groups of 8 as uint64's (pad the final one with 0 bytes if needed), and XORing the results together. But I think the alternative approach below will make the fuzzer more efficient.)

Reply:
Good point. But also xor is commutative, so each 8 bytes order can be switched without changing the output.

Reply:
Yes, that's true-- but if the input is longer than 8 bytes, then there must be many inputs which result in the same 8 byte seed, that's unavoidable. XOR just accomplishes the same sort of thing without multiplication of big numbers.